anomalib/models/cfa/config.yaml

```diff
@@ -0,0 +1,112 @@
+dataset:
+  name: mvtec #options: [mvtec, btech, folder]
+  format: mvtec
+  path: ./datasets/MVTec
+  category: bottle
+  task: segmentation
+  image_size: 224
+  train_batch_size: 4
+  test_batch_size: 4
+  num_workers: 8
+  transform_config:
+    train: null
+    val: null
+  create_validation_set: false
+  tiling:
+    apply: false
+    tile_size: null
+    stride: null
+    remove_border_count: 0
+    use_random_tiling: False
+    random_tile_count: 16
+
+model:
+  name: cfa
+  backbone: wide_resnet50_2
+  gamma_c: 1
+  gamma_d: 1
+  lr: 1e-3
+  weight_decay: 5e-4
+  amsgrad: true
+  early_stopping:
+    patience: 5
+    metric: pixel_AUROC
+    mode: max
+  normalization_method: min_max # options: [null, min_max, cdf]
+
+metrics:
+  image:
+    - AUROC
+  pixel:
+    - AUROC
```

Comment:
To be in line with other models it would be good to add both image- and pixel-level F1 here